internal/cli/branding_texts_test.go

```diff
@@ -91,50 +90,3 @@ func TestBrandingTextsShowCmd(t *testing.T) {
 		})
 	}
 }
-
-func TestBrandingTextsUpdateCmd(t *testing.T) {
```

Comment:
I couldn't make this format for the tests work due to the fact that we open the editor. While we can mock the API calls I couldn't find a clean way to mock the editor opening. Happy to hear suggestions if there's an easy approach. 🌱 

Otherwise we'll have to rework a bit dependencies so we attach them to the CLI through an interface and mock those parts.

Reply:
Can we pass the input via stdin?

Reply:
I wasn't able to cuz the command runs instantly, doesn't block to wait for my input.